### ISSUE TITLE
[UI/UX] Some Suggestions for the Home Page

### ISSUE THREAD

**Issue by Heus-Sueh** [Chocobozzz/PeerTube#5829]
### Describe the problem to be solved

I think the home page is a bit messy and poorly organized in my opinion some elements are placed in unusual places.

## Before:
![image](https://github.com/Chocobozzz/PeerTube/assets/98482469/a4f7b4ee-a826-4f35-a819-0e971c41c8a8)

### Describe the solution you would like

I have a few solutions for the UI/UX:

- move the user profile to the top right because it's more natural and the default for almost any site of this type
- move the home and other tabs to the top because it doesn't make sense for it to be in the middle.
- move the notifications button to be on the left side of the user profile

## After:
![New UI](https://github.com/Chocobozzz/PeerTube/assets/98482469/7202ce89-8e18-4998-8023-6ece5166e853)

for now I only had these ideas if you have more ideas please share them here :+1:

**Comment by tio-trom** [comment]
For one tilvids is using an old peertube version 4.3.1 and second admins are the ones creating these pages. Heres' how my instance looks like https://videos.trom.tf/home:

![2023-06-24_01-58](https://github.com/Chocobozzz/PeerTube/assets/38043315/0e215bb3-1ce6-46b7-a171-3285c59a095e)

So I would say the improvement should be in terms of offering templates via the Peertube admin dashboard for the homepage.

**Comment by Heus-Sueh** [comment]
> For one tilvids is using an old peertube version 4.3.1 and second admins are the ones creating these pages. Heres' how my instance looks like https://videos.trom.tf/home:
> 
> ![2023-06-24_01-58](https://user-images.githubusercontent.com/38043315/248431032-0e215bb3-1ce6-46b7-a171-3285c59a095e.png)
> 
> So I would say the improvement should be in terms of offering templates via the Peertube admin dashboard for the homepage.

I agree, how much you can modify the UI is very little, you can only change a few colors here and there.

**Comment by Chocobozzz** [comment]
I'm closing this inactive discussion, some elements were moved in the header with the new PeerTube theme.
